### PR TITLE
Implement new flow of offer creation

### PR DESCRIPTION
**Pull request by NicolasMahe**
### Project organization

- Related https://github.com/liteflow-labs/backend/issues/2407
- Dependant on https://github.com/liteflow-labs/backend/pull/2408

### Description

- update `useCreateOffer` hook with new `createOfferSignature` and updated `createOffer` mutations

Build is falling because need new version of @nft/api-graphql. Can run locally.

### Checklist

- [x] Update related changelogs

### CODE REVIEW

File: packages/hooks/src/useCreateOffer.ts [hunk 1]
```diff
@@ -9,43 +9,60 @@ import { OfferType, TransactionFragment } from './graphql'
 import { convertTx } from './utils/transaction'
 
 gql`
-  mutation CreateOffer(
-    $input: OfferInputBis!
+  query FetchAssetAndCurrency(
```
Comment (antho1404) [inline packages/hooks/src/useCreateOffer.ts]: Let's rename that into fetch approval or something related to the approval

**Review by antho1404 — APPROVED** [review]
One tiny thing about naming but not a blocker

Let's just merge this when the new API is deployed

File: packages/hooks/src/useCreateOffer.ts (hunk 1 above)
Comment (NicolasMahe) [reply packages/hooks/src/useCreateOffer.ts]: This query is going to be removed with https://github.com/liteflow-labs/liteflow-js/pull/117.
The two PRs 119 and 117 modify the same queries. A bit hard to follow...
Will be better when new version of backend and api-graphql is released an we can merge everything.

Comment (antho1404) [reply packages/hooks/src/useCreateOffer.ts]: let's not touch it then

**Review by antho1404 — APPROVED** [review]
@NicolasMahe I merged the conflict but it was a bit tricky. Please have another look, it's good to go on my side (when API is released)
